pkg/network/java/testutil/JavaClientSimulator/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Base image with Java installed
+FROM openjdk:11
+MAINTAINER val
+
+# Set the working directory inside the container
+WORKDIR /app
+
+COPY target/JavaClientSimulator-1.0.jar app.jar
+ENTRYPOINT ["sh", "-c", "java -jar app.jar -c \"$JAVA_TEST_CLIENT\" -u \"$JAVA_TARGET_URL\""]
```

Comment:
I don't see any `app.jar` in that PR
Do you want to add it? Besides that, please add documentation about compiling the jar, building and running the docker

Reply:
the app.jar is copied into a docker, the original jar file is called `JavaClientSimulator-1.0.jar` and can be built using `mvn package` on JavaClientSimulator directory (it will be placed under the `target` sub-folder)